scripts/run_frontend_tests.py

```diff
@@ -29,6 +29,9 @@
 from . import common
 from . import install_third_party_libs
 
+# This is a relative path from the oppia/ folder.
```

Comment:
Explain why a relative path is needed (mention that dtslint seems to prepend the cwd to the path passed in, even if its an absolute path that has already been constructed with os.getcwd())

Reply:
Done